arch/risc-v/src/mpfs/mpfs_dma_alloc.c

```diff
@@ -0,0 +1,94 @@
+/****************************************************************************
```

Comment:
I do not see how this file is going to be involved in compilation process

Reply:
Forgot to add it for the nuttx build system (I use a custom system myself). Added now.